src/Tizen.Content.Download/Tizen.Content.Download/CacheManager.cs

```diff
@@ -0,0 +1,162 @@
+/*
+* Copyright (c) 2016 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.ComponentModel;
+
+namespace Tizen.Content.Download
+{
+    /// <summary>
+    /// The CacheManager class provides the functions to manage cache properties.
+    /// </summary>
+    /// <since_tizen> 7.5 </since_tizen>
+    static class CacheManager
+    {
+
+        /// <summary>
+        /// Clears the cache.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when it is failed due to an invalid operation.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when a permission is denied.</exception>
+        /// <exception cref="NotSupportedException">Thrown when not supported.</exception>
+        static public void ResetCache()
+        {
+            int ret = Interop.Download.ResetDownloadCache();
+            if (ret != (int)DownloadError.None)
+            {
+                DownloadErrorFactory.ThrowException(ret, "Failed to reset cache");
+            }
+        }
+
+        /// <summary>
+        /// Max size(MB) of the cache.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
```

Comment:
standard meaning of ArgumentException is that there is something wrong with the value of method argument, this is a property. If that exception can actually be thrown, then let it stay, but the description could be changed to sth like say e.g. that setting to invalid value was rejected. Is there something like invalid value? I guess any value would work.

oh, it's not a public API, never mind then

Reply:
Updated to "Thrown when it is failed due to setting an invalid value."